daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1141,6 +1153,7 @@ private[lf] object SBuiltin {
               onLedger.committers,
               { case Versioned(_, V.ContractInstance(actualTmplId, arg, _)) =>
                 checkTemplateId(actualTmplId) {
+                  machine.pushKont(KWrapInterface(machine, actualTmplId))
```

Comment:
hm, it seems like we should be able to use `SBToInterface` instead of a custom continuation but I can’t quite figure out how to do that in a way that isn’t more annoying :thinking:

Reply:
I think `KPap(PBuiltin(SToInterface(actualTmplId)), new ArrayList(), 1)` works. Not sure if that’s better though. @remyhaemmerle-da wdyt?

Reply:
The continuation will disappear with #12797, so I am not very worry. 
I will for @cocreature version as it will be faster to clean.

Reply:
will do, thanks for the suggestion @cocreature @remyhaemmerle-da!